src/main/java/app/Main.java

```diff
@@ -96,9 +92,11 @@ public static void main(String[] args) {
         FileTokenDataAccessObject tokenDataAccessObject = new FileTokenDataAccessObject();
         UserPreferencesFactory userPreferencesFactory = new UserPreferencesFactory();
         ServerProfileDataAccessObject profileDataAccessObject = new ServerProfileDataAccessObject(tokenDataAccessObject, userPreferencesFactory);
+        PasswordValidatorService passwordValidatorService = new PasswordValidatorService();
```

Comment:
Implementation of the Validator Services allow for a good implementation of the single responsibility principle (SOLID)!